### PR TITLE
Migration from Ant to Maven

### PR DESCRIPTION
**Pull request by kronenthaler**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /dist/
 /nbproject/private/
```
Comment (dktcoding) [inline .gitignore]: `/dist/` and `/nbproject/private/` are no longer necessary.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (dktcoding) [inline pom.xml]: I'm getting an encoding warning, shouldn't this be `UTF-8`?.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>kronenthaler</groupId>
+    <artifactId>libai</artifactId>
```
Comment (dktcoding) [inline pom.xml]: The [main class](https://maven.apache.org/shared/maven-archiver/examples/classpath.html#Make) is not set, it seems like a good idea if it runs the examples when excecuted.

File: pom.xml (hunk 2 above)
Comment (kronenthaler) [inline pom.xml]: Where do you get the warning? i'm not getting anything but some code warnings

Comment (dktcoding) [inline pom.xml]: I run `mvn jar:jar` and `mvn test` on Fedora 25 and got an "Warning using system encoding".
The error wasn't present in either Ubuntu Jaunty (Travis CI) or Xenial (perhaps it was one of Fedora's "quirks")

**Pull request merged**